src/test/java/com/wikia/webdriver/testcases/globalnavigationbar/Navigating.java

```diff
@@ -48,14 +48,14 @@ public void testTVHubLink() {
   }
 
   @Test(groups = {"communityCentralLinkClickOnDeCommunityOpensDeCommunityCentral"})
-  @Execute(onWikia = "de.gta")
+  @Execute(onWikia = "gta", language = "de")
   public void testCommunityCentralLinkOnDeCommunity() {
     new HomePage()
         .getGlobalNavigation()
         .clickCommunityCentralLink();
 
     Assertion.assertEquals(driver.getCurrentUrl(),
-                           urlBuilder.getUrlForPage("de.community", "/wiki/Community_Deutschland"));
+            urlBuilder.getUrlForPath("community", "de", "/wiki/Community_Deutschland"));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "community" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)